docs/dev/architecture/workflow.md

```diff
@@ -0,0 +1,31 @@
+# Transformation workflow
+
+This document describes the current transformation workflow.
+Older versions are described afterwards.
+
+## Current Flow
```

Comment:
This method seems to be the most viable one, however the problem with the platform dependant Shell Scripts has to be adressed. Maybe we can just supply a Batch script and a Shell script to guarantee platform independance

Reply:
Please reconsider the usage of Cygwin on windows.
Do you consider it a deal breaker to request windows user to install Cygwin for deployment? I estimate the percentage of dev/ops running Windows but not Cygwin on their work machine approaches 0.